addons/hr_holidays/report/hr_leave_employee_type_report.py

```diff
@@ -102,3 +103,8 @@ def action_time_off_analysis(self):
                 'group_expand': True,
             }
         }
+
+    def _search_allocation_count(self, operator, value):
+        employees = self.env['hr.employee'].search([])
+        employees_with_remaining_leaves = employees.filtered(lambda e: e.allocation_count > 0)
```

Comment:
You should consider the `operator` and `value` args:

```suggestion
        employees_with_remaining_leaves = employees.filtered_domain([('allocation_count', operator, value)])
```